python/dlisio/plumbing/frame.py

```diff
@@ -379,8 +399,52 @@ def curves(self):
         >>> curves = frame.curves()
         >>> pdcurves = pd.DataFrame(curves, index=curves[f.index])
         >>> pdcurves.index.name = f.index
+
+        By default the returned np.ndarray have column-names that reflects the
+        mnemonics of each Channel
+
+        >>> frame.channels
+        [Channel(TDEP), Channel(TDEP), Channel(GR)]
+
+        >>> curves = frame.curves()
+        >>> curves.dtype.names
+        ('FRAMENO', 'TDEP', 'TDEP', 'GR')
+
+        Sometimes a Frame can contain multiple Channels with the same
+        name/mnemonic, in that case the labels are augmented with the origin and
+        copynumber:
+
+        >>> frame.channels
+        [Channel(TDEP), Channel(TDEP), Channel(GR)]
+
+        >>> curves = frame.curves()
+        >>> curves.dtype.names
+        ('FRAMENO', 'TDEP.0.0', 'TDEP.0.1', 'GR')
+
+        This augmentation is customizable by changing :attr:`dtype_fmt`. See
+        :attr:`dtype`. However, in some cases even this is not enough to make
+        unique labels for the columns. That would be a clear violation of the
+        dlis spec and :attr:`curves` will raise an ValueError:
+
```

Comment:
This augmentation is customizable by changing :attr:`dtype_fmt`. See :attr:`dtype`. Some frames have multiple instance of the same channel or mulitple channels where name, origin and copynumber are identical. This is a clear violation of the dlis spec and :attr:`curves()` will raise an ValueError by default.